docs/snippets/all/archetypes/series_lines_style.py

```diff
@@ -9,12 +9,12 @@
 # Set up plot styling:
 # They are logged as static as they don't change over time and apply to all timelines.
 # Log two lines series under a shared root so that they show in the same plot by default.
-rr.log("trig/sin", rr.SeriesLine(color=[255, 0, 0], name="sin(0.01t)", width=2), static=True)
-rr.log("trig/cos", rr.SeriesLine(color=[0, 255, 0], name="cos(0.01t)", width=4), static=True)
+rr.log("trig/sin", rr.SeriesLines(colors=[255, 0, 0], names="sin(0.01t)", widths=2), static=True)
+rr.log("trig/cos", rr.SeriesLines(colors=[0, 255, 0], names="cos(0.01t)", widths=4), static=True)
```

Comment:
wondering whether I should put those in the same entity now for illustration 🤔 
practically though separate entities make more sense here